src/Build/BackEnd/Components/RequestBuilder/TaskBuilder.cs

```diff
@@ -424,9 +424,11 @@ private async Task<WorkUnitResult> ExecuteBucket(TaskHost taskHost, ItemBucket b
                     // We need to find the task before logging the task started event so that the using task statement comes before the task started event
                     IDictionary<string, string> taskIdentityParameters = GatherTaskIdentityParameters(bucket.Expander);
                     TaskRequirements? requirements = _taskExecutionHost.FindTask(taskIdentityParameters);
+                    AssemblyName taskAssemblyName = _taskExecutionHost.FindTaskAssemblyName(taskIdentityParameters);
```

Comment:
nit: Can the return type of `FindTask` be changed instead, so that requirements and assembly name are returned from one call?

Reply:
Done